pkg/hub/deployer/deployer.go

```diff
@@ -308,6 +334,198 @@ func (deployer *Deployer) handleSubscription(sub *appsapi.Subscription) error {
 	return nil
 }
 
+func (deployer *Deployer) handleAggregateStatus(sub *appsapi.Subscription) error {
```

Comment:
I think we could move this to `aggregatestatus.go`, since we can get all the status from the listers.

The aggregatestatus controller can be invoked in deployer.

Reply:
> I think we could move this to `aggregatestatus.go`, since we can get all the status from the listers.
> 
> The aggregatestatus controller can be invoked in deployer.

Yes, it can be done in code implementation. In fact, putting handleAggregateStatus in aggregatestatus.go or deployer.go has the same effect. Considering that there will be more iterations and more code in the subsequent status aggregation, we can try to put it in aggregatestatus.go.